src/app/(sidebar)/transaction/sign/components/Overview.tsx

```diff
@@ -0,0 +1,286 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { Card, Icon, Text, Button } from "@stellar/design-system";
+import { FeeBumpTransaction } from "@stellar/stellar-sdk";
+
+import { FEE_BUMP_TX_FIELDS, TX_FIELDS } from "@/constants/signTransactionPage";
+
+import { useStore } from "@/store/useStore";
+
+import { transactionSigner } from "@/helpers/transactionSigner";
+
+import { validate } from "@/validate";
+
+import { Box } from "@/components/layout/Box";
+import { MultiPicker } from "@/components/FormElements/MultiPicker";
+import { TextPicker } from "@/components/FormElements/TextPicker";
+import { WithInfoText } from "@/components/WithInfoText";
+import { ValidationResponseCard } from "@/components/ValidationResponseCard";
+import { XdrPicker } from "@/components/FormElements/XdrPicker";
+
+const MIN_LENGTH_FOR_FULL_WIDTH_FIELD = 30;
+
+export const Overview = () => {
+  const { network, transaction } = useStore();
+  const { sign, updateSignActiveView, updateSignedTx, resetSign } = transaction;
+
+  const [secretInputs, setSecretInputs] = useState<string[]>([""]);
+  const [signedTxSuccessMsg, setSignedTxSuccessMsg] = useState<string>("");
+
+  // @TODO bip path
+  const [bipPath, setBipPath] = useState<string>("");
```

Comment:
Values that we want to persist when the page is refreshed, we should save in the store.

Reply:
i haven't started working on bipPath! I will do that in the next pr